### DISCUSSION TITLE
Develop with me

### DISCUSSION THREAD

**Discussion by blueedgetechno** [blueedgetechno/win11React#19] [Ideas]
Try to program any of this or anything you like

- ~Functional Spotify with [unoffical jiosaavn api](https://github.com/cyberboysumanjay/JioSaavnAPI).~
- ~File explorer with proper file node structure.~
- ~Lock screen and password screen.~
- ~Drag and resize windows.~
- ~Themes and background.~

You can then make a pull request. Try to follow the code and file structure of the project.

**Comment by andrewstech** [comment]
> Try to program any of this or anything you like
> 
> * Functional Spotify with [unoffical jiosaavn api](https://github.com/cyberboysumanjay/JioSaavnAPI).
> * File explorer with proper file node structure.
> * Lock screen and password screen.
> * Drag and resize windows.
> * Themes and background.
> 
> You can then make a pull request. Try to follow the code and file structure of the project.

Think we need some new targets.

**Comment by ghost** [comment]
Some ideas.
-Settings app
![settings-app-r](https://user-images.githubusercontent.com/77000356/151996639-9098d853-ce1b-4024-a952-78c38f29729a.jpg)

-Shut down screen
![download (1)](https://user-images.githubusercontent.com/77000356/151996100-0e27362f-8e1c-4188-a1b9-1f90166b6963.png)
-Control panel
![Windows-11-Control-Panel](https://user-images.githubusercontent.com/77000356/151996657-ba1a830c-99a2-45d0-b463-6f8b39443966.png)

-OOBE
![4cca397f-2754-4085-a425-14135f86cbee](https://user-images.githubusercontent.com/77000356/151996267-a5fc48a4-f696-441b-afb9-643c05e05382.png)
-Functioning recycle bin.

**Reply by briantheman** [reply]
Good IDEA. However settings are being worked on https://win11-settings.yashash.me/

OWO
